From 060ba5c07a37d309bd3e85bb5096bf29feebce3c Mon Sep 17 00:00:00 2001
From: Ferdinand Xu <cheng.a.xu@intel.com>
Date: Wed, 19 Oct 2016 01:38:17 +0800
Subject: [PATCH 0796/1164] CDH-46233: HIVE-15000: Remove addlocaldriverjar,
 and addlocaldrivername from command line help
 (Peter Vary, reviewed by Ferdinand Xu)

(cherry picked from commit 399af4663f9f4a4ec85b48523b0bc54e5804b87a)

Change-Id: Idb2e154f46754c0b222e81f314c7dc81963e99da
---
 beeline/src/main/resources/BeeLine.properties |    2 --
 1 file changed, 2 deletions(-)

diff --git a/beeline/src/main/resources/BeeLine.properties b/beeline/src/main/resources/BeeLine.properties
index 421bcf7..b93d002 100644
--- a/beeline/src/main/resources/BeeLine.properties
+++ b/beeline/src/main/resources/BeeLine.properties
@@ -188,8 +188,6 @@ cmd-usage: Usage: java org.apache.hive.cli.beeline.BeeLine \n \
 \  --delimiterForDSV=DELIMITER     specify the delimiter for delimiter-separated values output format (default: |)\n \
 \  --isolation=LEVEL               set the transaction isolation level\n \
 \  --nullemptystring=[true/false]  set to true to get historic behavior of printing null as empty string\n \
-\  --addlocaldriverjar=DRIVERJARNAME Add driver jar file in the beeline client side\n \
-\  --addlocaldrivername=DRIVERNAME Add drvier name needs to be supported in the beeline client side\n \
 \  --help                          display this message\n \
 \n \
 \  Example:\n \
-- 
1.7.9.5

